src/components/configuration/HttpRequestActionConfig.vue

```diff
@@ -2,7 +2,17 @@
   <ExpansiblePanel no-top-divider no-bottom-divider :is-expanded="!interfaceStore.isOnPhoneScreen">
     <template #title>HTTP Request Actions</template>
     <template #info>
-      <p>View, manage, and create HTTP request actions.</p>
+      <li>View, manage, and create HTTP request actions.</li>
+      <li>You can use those actions to call external APIs, like servers, vehicles, cameras, etc.</li>
+      <li>
+        You can use mustaches to refer to data-lake variables on the URL and body. You can also use data-lake variables
```

Comment:
```suggestion
        You can use <code>{{ variable-id }}</code> to refer to data-lake variables on the URL and body. You can also use data-lake variables
```

Reply:
I was trying to make that work yesterday but I couldn't prevent the mustaches from trying to render the content inside. Found a solution today:

```
You can use <code>&#123;&#123; variable-id &#125;&#125;</code> to refer to [...]
```

which renders to:

<img width="436" alt="image" src="https://github.com/user-attachments/assets/2a199421-fe11-4023-8520-59fd70264eba" />

Reply:
Hmm, presumably [`v-pre`](https://www.w3schools.com/vue/ref_v-pre.php) would do it, but this works too I suppose 🤷‍♂️ :-)